sdk/src/main/java/com/bugsnag/android/FileStore.java

```diff
@@ -16,8 +17,11 @@
 
     @NonNull
     protected final Configuration config;
+
     @Nullable
-    final String storeDirectory;
+    final String oldDirectory;
+
+    File storageDir;
```

Comment:
Why should this property be renamed?

Reply:
I wanted it to be clear that there is a difference between the old directory we previously used, and the client-specific directory that we will use in future. Maybe the nomenclature is a bit off here?